vocab.py

```diff
@@ -0,0 +1,22 @@
+# standard libraries
+from enum import auto, Enum
+
+
+# Enum(s)
+
+# Each job name should be defined in ValidJobName.
+# NAME_OF_JOB = 'path.to.method'
```

Comment:
What do you think of moving comments like these into docstrings?

Reply:
I moved them, let me know if you see any issues with them.